src/monmove.cpp

```diff
@@ -824,6 +840,15 @@ int monster::attack_at(int x, int y) {
 
 int monster::move_to(int x, int y, bool force)
 {
+    //Allows climbing monsters to move on terrain with movecost <= 0
+    if (g->m.has_flag_ter_or_furn("CLIMBABLE", x, y) && has_flag(MF_CLIMBS) && g->npc_at(x, y) == -1 && g->mon_at(x, y) == -1) {
+            if (g->u.sees( *this )){
+                    add_msg(_("The %s climbs over the %1$s."), name().c_str(),
+                            g->m.has_flag_ter_or_furn("CLIMBABLE", x, y) ? g->m.tername(x, y).c_str() : g->m.furnname(x, y).c_str());
```

Comment:
I should just use has_flag_ter and I need to correct that in a few other places elsewhere in this PR.